### PR TITLE
python3-Pyro4: update to 4.81.

### PR DESCRIPTION
**Pull request by pfpulux**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/python3-Pyro4/template [hunk 1]
```diff
@@ -1,20 +1,31 @@
...
 wrksrc="${pkgname#*-}-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-setuptools python3-serpent"
+# checkdepends="python3-serpent python3-msgpack python3-dill python3-cloudpickle"
+checkdepends="python3-serpent"
 short_desc="Distributed Object Middleware (IPC/RPC) for Python3"
 maintainer="pulux <pulux@pf4sh.de>"
 license="MIT"
 homepage="https://pythonhosted.org/Pyro4/"
```
Comment (ahesford) [inline srcpkgs/python3-Pyro4/template:14]: Homepage is dead, please replace with https://pyro4.readthedocs.io/

File: srcpkgs/python3-Pyro4/template [hunk 2]
```diff
@@ -1,20 +1,31 @@
...
 
 conflicts="python-Pyro4>=0"
 
+do_check() {
+	# Tests depend on python3-dill and python3-cloudpickle, which aren't packaged
+	rm -f tests/PyroTests/test_daemon.py tests/PyroTests/test_serialize.py
+	# Tests can't get addr
+	rm -f tests/PyroTests/test_naming.py tests/PyroTests/test_naming2.py
+	rm -f tests/PyroTests/test_socket.py tests/PyroTests/test_util.py
+	# PYTHONPATH=src python3 -E -Wall -tt -bb tests/run_testsuite.py
```
Comment (ahesford) [inline srcpkgs/python3-Pyro4/template:26]: `do_check` isn't actually doing anything after remove those that don't work.

**Review by ahesford — CHANGES_REQUESTED** [review]

File: srcpkgs/python3-Pyro4/template (hunk 2 above)
Comment (pfpulux) [reply srcpkgs/python3-Pyro4/template:26]: yes I know, but the run_testsuite.py depends on Pyro4 and this was not found in the PR-Test Env. How can we handle this?

File: srcpkgs/python3-Pyro4/template (hunk 1 above)
Comment (pfpulux) [reply srcpkgs/python3-Pyro4/template:14]: will do

File: srcpkgs/python3-Pyro4/template [hunk 3]
```diff
@@ -1,20 +1,31 @@
...
 pkgname=python3-Pyro4
-version=4.80
-revision=4
+version=4.81
+revision=1
 wrksrc="${pkgname#*-}-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-setuptools python3-serpent"
+# checkdepends="python3-serpent python3-msgpack python3-dill python3-cloudpickle"
```
[the remark this answers is not in the document]
Comment (pfpulux) [reply srcpkgs/python3-Pyro4/template:9]: will do

File: srcpkgs/python3-Pyro4/template (hunk 2 above)
Comment (ahesford) [reply srcpkgs/python3-Pyro4/template:26]: If getting tests to work without installing the package, I would delete `checkdepends` and the `do_check` altogether, set `make_check="no"` and leave a comment to the effect that tests don't work unless the package is installed (cf. `python3-numpy`).

**Pull request closed without merging**